js/views/chatHeadsVw.js

```diff
@@ -18,6 +18,18 @@ module.exports = baseVw.extend({
         this.createChatHead(md).render().el
       );
     });
+
+    this.$chatHeadsContainer = $('.chatConversationHeads');
+    this.$headContainer = $('<div />');
+
+    this.showPerScroll = 10;
```

Comment:
Good point. We could do something with calculating the height and figuring out how many would fit in that height, but if we have to load one and pull the height from the DOM that could get slow. 

Possibly it could stay at 10, and detect if the total height of the wrapper div is equal to the height of $headContainer, and if it's too short, load another 10.